examples/svelte-apollo/src/AddArticles.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+  import { mutation } from 'svelte-apollo'
+  import { gql } from '@apollo/client'
+
+  const ARTICLE_LIST = gql`
+    query {
+      article(order_by: [{ title: asc }]) {
+        title
+      }
+    }
+  `
+  const ADD_ARTICLE = gql`
+    mutation ($title: String!, $content: String!, $author_id: Int!) {
+      insert_article(objects: [{ title: $title, content: $content, author_id: $author_id }]) {
```

Comment:
We can use a generic `$article: articles_insert_input!` here instead.

Like this (untested code):

```graphql
    mutation ($article: articles_insert_input!) {
      insert_article(objects: [$article]) {
```

Reply:
Done, only testing is left.  will test this code as well shortly